test262/per-file/main.js

```diff
@@ -0,0 +1,225 @@
+"use strict";
+
+// (() => {
```

Comment:
These can probably be removed (here & at the end). I added an IIFE in the other file to not pullute the global object but honestly who cares, having to indent everything is equally annoying and it's not like this code will ever be used on another website we don't "own" `window`.